code/api/src/modules/user/tests/userMutations.test.js

```diff
@@ -14,4 +16,54 @@ describe('user mutations', () => {
       })
     )
   })
+
+  beforeAll(async () => {
+    const tom = await models.User.create({
+      name: 'Tom',
+      email: "tomtom@tom.com",
+      password: "12345",
+      role: null,
+      style: "Radical"
+    });
+    const steve = models.User.create({
+      name: 'Steve',
+      email: "stylesssteve@aww.com",
+      password: "12345",
+      role: null,
+      style: null
+    })
+  })
+
+  afterAll(async () => {
+    await models.User.destroy({
+      where: {
+        email: 'tomtom@tom.com'
+      }
+    });
+    models.User.destroy({
+      where: {
+        email: 'stylesssteve@aww.com'
+      }
+    });
+  })
+
+  it('Can successfully update a user style from null', async (done) => {
+    const response = await request(server)
+    .post('/')
+    .send( {query: 'mutation {userStyleUpdate(email: "stylesssteve@aww.com", style: "Bohemian") {email, style}}'} )
```

Comment:
Another really hard won little line of code! Interesting how GraphQL takes in queries in string format.